src/217-Contains-Duplicate/217._Contains_Duplicate.py

```diff
@@ -0,0 +1,14 @@
+class Solution:
+    def containsDuplicate(self, nums):
+        """
+        Given an array of integers, find if the array contains any duplicates.
+
+        Args:
+            nums(List[int]): the name of the array of integers
+
+        Returns:
+            A bool value(bool)
+        """
+        if len(nums) > len(set(nums)):
+            return True
+        return False
```

Comment:
Why don't `return len(nums) > len(set(nums))` directly here?

Reply:
OK, this will be more concise